builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -479,11 +517,11 @@ def updateSwaggerConfig() {
 			sh "sed -i -- 's/{service_name}/${service_config['service']}/g' swagger/swagger.json"
 			sh "sed -i -- 's/{domain}/${service_config['domain']}/g' swagger/swagger.json"
 
-			def region = "${configLoader.AWS.REGION}"
+			def region = "${service_config.region}"
+			def accountObject = utilModule.getAccountInfo(service_config)
+			def roleARN = accountObject.IAM.PLATFORMSERVICES_ROLEID.replaceAll("/", "\\\\/")
 
-			def roleARN = configLoader.AWS.PLATFORMSERVICES_ROLEID.replaceAll("/", "\\\\/")
-
-			// TODO: the below couple of statements will be replaced with regular expression in very near future;
+      // TODO: the below couple of statements will be replaced with regular expression in very near future;
```

Comment:
Fix Formatting

Reply:
the formatting is correct in the file. Not sure why is it showing different in the diff